### DISCUSSION TITLE
Add another set of coordinate values in the same point entity in QField (line start and end)

### DISCUSSION THREAD

**Discussion by Plant15** [opengisch/QField#2959] [Ideas]
In an attemp to add another set of x,y coordinate to QField (mark the start and the end of a line feature with points) I have created another set of "lat" "long" fields linked to a checkbox as indicated here :[QGIS 3.16 Use change state of a checkbox as a trigger for updating another field](https://gis.stackexchange.com/questions/427599/qgis-3-16-use-change-state-of-a-checkbox-as-a-trigger-for-updating-another-field)

Unfortunately the formula in the field : if("State=end", $x, 'NULL') didn't add a new coordinate, the one of my position after moving from the start (or the position of the cursor distant to the start) but instead the same value as the entity location.

To resume, at my sense, QField don't allow to create line informations based on an entity with 2 locations. I am surely not the only one working in biological sciences or any other professional discipline making forms on the field and in need of this function. It would be kind of a $x and $y command but for the cursor position. This app is fantastic, thank you.

**Comment by nirvn** [comment]
If what you are trying to achieve works in QGIS but fails in QField, you should attach a small test project that demonstrates what you are trying to achieve. The description alone is a bit confusing (to me at least).
